content/python/concepts/random-module/terms/uniform/uniform.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.uniform()'
+Description: 'Returns a pseudo-random floating-point number between two given numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Random'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.uniform()` method takes two numbers as arguments and returns a pseudo-random floating-point number between them. The result is inclusive of the first value, and possibly inclusive of the second value, depending on rounding.
```

Comment:
```suggestion
The `.uniform()` method takes two numbers as arguments and returns a pseudo-random floating-point number between them. The result is inclusive of the first value, and possibly inclusive of the second value, depending on rounding.
```

Reply:
One last thing! To keep everything consistent across the rest of the random mod methods 🙏